dashboard/config/routes.rb

```diff
@@ -18,7 +18,7 @@ module OPS
   get '/dashboardapi/terms-and-privacy', to: "home#terms_and_privacy"
   get '/dashboardapi/teacher-announcements', to: "home#teacher_announcements"
 
-  get '/teacher', to: 'home#teacher_homepage'
+  get '/courses', to: 'home#courses'
```

Comment:
@aoby Is there a simpler way to set up a route for a basic page (that theoretically is handled by the `home_controller`)?